### PR TITLE
docs: add CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by jsjoeio**
As part of our effort to make the code-server community more inclusive, I'd like to propose adding a `CODE_OF_CONDUCT`

### CODE REVIEW

File: docs/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at joe@coder.com. All
```
Comment (jsjoeio) [inline docs/CODE_OF_CONDUCT.md:61]: Ideally we would have `opensource@coder.com` - cc @bpmct ?

Comment (bpmct) [reply docs/CODE_OF_CONDUCT.md:61]: I agree. For now, we can use `contact@coder.com`.

**Review by bpmct — CHANGES_REQUESTED** [review]

File: docs/CODE_OF_CONDUCT.md (hunk 1 above)
Comment (jsjoeio) [inline docs/CODE_OF_CONDUCT.md:61]:
```suggestion
reported by contacting the project team at contact@coder.com. All
```

File: docs/CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at contact@coder.com. All
```
Comment (jsjoeio) [inline docs/CODE_OF_CONDUCT.md:61]:
```suggestion
reported by contacting the project team at opensource@coder.com. All
```

**Review by bpmct — APPROVED** [review]

**Pull request merged**